src/modules/jcms_rest/tests/assets/subjects.sql

```diff
@@ -1,458 +0,0 @@
-# ************************************************************
-# Sequel Pro SQL dump
```

Comment:
Once you merge this in you should squash the commits so that this SQL isn't in the commit history as it was just a temporary measure to demonstrate the tests would pass if the subject taxonomy terms were in place.